node/sc/main_bridge_handler.go

```diff
@@ -75,6 +83,32 @@ func (mbh *MainBridgeHandler) HandleSubMsg(p BridgePeer, msg p2p.Msg) error {
 	return nil
 }
 
+func (mbh *MainBridgeHandler) handleCallMsg(p BridgePeer, msg p2p.Msg) error {
+	// Decode p2p message
+	// TODO-Klaytn-ServiceChain: check msg.Size is enough.
```

Comment:
This TODO means different one with ErrRPCBufferOverflow?

Reply:
Yes. I put in an error log to see if the ToDo item is really meaningful and I will see the log taken during the test.